apps/services/auth-public-api/test/setup.ts

```diff
@@ -0,0 +1,71 @@
+import {
```

Comment:
This is a file that we want to make public by moving it to a library (or root) and have jest.config reference it.

Reply:
This sets up a test server with database integration (suitable for integration/functional testing)